app/src/main/java/com/example/yauheni_skarakhodau/epamlearnwork/MyRecyclerAdapter.java

```diff
@@ -0,0 +1,71 @@
+package com.example.yauheni_skarakhodau.epamlearnwork;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.List;
+
+public class MyRecyclerAdapter extends RecyclerView.Adapter<MyRecyclerAdapter.MyViewHolder> {
+
+    private List<String> items;
+
+    @NonNull
+    @Override
+    public MyViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        MyViewHolder holder = new MyViewHolder(new TextView(viewGroup.getContext()));
+        return holder;
```

Comment:
I would like you suggest this solution

final View view = new TextView(viewGroup.getContext());
        final RecyclerView.ViewHolder holder = new MyViewHolder(view);
        view.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(final View view) {
                final int adapterPosition = holder.getAdapterPosition();
                if (adapterPosition != RecyclerView.NO_POSITION) {
                    clicker.onClick(items[adapterPosition]);
                }
            }
        });
        return (MyViewHolder) holder;